src/output.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const { OUTPUT_MESSAGE, bullsMessage, cowsMessage } = require('./constants');
+
+const output = ({ bulls, cows }) => {
+  const message = bullsMessage(bulls) + cowsMessage(cows);
+
+  if (!message) {
+    return OUTPUT_MESSAGE.noMatch;
+  } else {
+    return message;
+  }
```

Comment:
Try to simplify the code where possible:
```suggestion
  if (!message) {
    return OUTPUT_MESSAGE.noMatch;
  }
  return message;
```